tests/test_other.py

```diff
@@ -2274,9 +2274,9 @@ def test_conftest_s_flag_passing(self):
         return 0;
       }
     ''')
+    cmd = [EMCC, '-s', 'ASSERTIONS', 'conftest.c', '-o', 'conftest']
     with env_modify({'EMMAKEN_JUST_CONFIGURE': '1'}):
-      cmd = [EMCC, '-s', 'ASSERTIONS', 'conftest.c', '-o', 'conftest']
-    output = self.run_process(cmd, stderr=PIPE)
+      output = self.run_process(cmd, stderr=PIPE)
```

Comment:
I guess that `env_modify` is not needed here since the filename (conftest.c) is enough of signal to run in this mode?  Maybe just remove the `with` and replac with a comment?